### PR TITLE
Update/ion kit 1.0.0

### PR DESCRIPTION
**Pull request by Fixstars-momoko**
### C++

- [x] tutorial 1 : API, parameters, got rid of optional input (Gain and ExposureTime)

### Python

- [x] tutorial 1 : API, parameters, got rid of optional input (Gain and ExposureTime)
- [x] tutorial 1 came2 : API, parameters, got rid of optional input (Gain and ExposureTime)

Note: Python binding issues are still under fixing at (https://github.com/fixstars/ion-kit/issues/206 and https://github.com/fixstars/ion-kit/issues/205)

### C++

- [x] tutorial 2 : enable to control Gain and ExposureTime

### Python

- [x] tutorial 2 : enable to control Gain and ExposureTime

### CODE REVIEW

File: cpp/src/tutorial1_display.cpp [hunk 1]
```diff
@@ -133,7 +98,6 @@ int main(int argc, char *argv[])
       int ret = video<uint16_t>(width, height, pixelformat, num_device);
     }
 
-
   }catch(std::exception& e){
```
Comment (Fixstars-iizuka) [inline cpp/src/tutorial1_display.cpp]: It should catch ion::Error additionally to std::exception.

File: cpp/src/tutorial1_display.cpp [hunk 2]
```diff
@@ -88,32 +54,31 @@ int video(int width, int height, std::string pixel_format, int num_device){
...
+    int loop_num = 100;
+    int coef =  positive_pow(2, num_bit_shift_map[pixel_format]);
     for (int i = 0; i < loop_num; ++i)
     {
-      pm.set(dispose_p, i == loop_num-1);
       // JIT compilation and execution of pipelines with Builder.
       try {
-          b.run(pm);
+        b.run();
       }catch(std::exception& e){
```
Comment (Fixstars-iizuka) [inline cpp/src/tutorial1_display.cpp]: We can delete try-catch from here and delegate to outermost try-catch block.

File: python/tutorial1_display.py [hunk 3]
```diff
@@ -92,18 +66,17 @@
...
     loop_num = 100
+    coef = pow(2, num_bit_shift)
 
     for x in range(loop_num):
-        port_map.set_u1(dispose_p, x==loop_num-1)
-
         # running the builder
-        builder.run(port_map)
+        builder.run()
         output_bytes = outputs[0].read(output_byte_size)
```
Comment (Fixstars-iizuka) [inline python/tutorial1_display.py]: `read` is no longer needed. You can create ion.Buffer passing numpy array  (`Buffer(array=numpy_array)`) and you can access data through numpy array without `ion.Buffer.read/write` operation

File: cpp/src/tutorial1_display.cpp (hunk 1 above)
Comment (Fixstars-momoko) [reply cpp/src/tutorial1_display.cpp]: fixed in 87d09cf

File: cpp/src/tutorial1_display.cpp (hunk 2 above)
Comment (Fixstars-momoko) [reply cpp/src/tutorial1_display.cpp]: fixed in  a85a0a5

File: python/tutorial1_display.py [hunk 4]
```diff
@@ -79,10 +52,7 @@
     outputs.append(Buffer(Type(TypeCode.Uint, depth_of_buffer, 1), output_size))
 
     # set I/O ports
-    port_map = PortMap()
-    port_map.set_buffer(gain_p, gains)
-    port_map.set_buffer(exposuretime_p, exposures)
-    port_map.set_buffer_array(output_p, outputs)
+    output_p[0].bind(outputs[0])
```
Comment (xinyuli1204) [inline python/tutorial1_display.py]: after pull https://github.com/fixstars/ion-kit/pull/207
you can use `output_p.bind(outputs)`

File: python/tutorial1_display.py (hunk 3 above)
Comment (Fixstars-momoko) [reply python/tutorial1_display.py]: fixed in 24b9ed0

**Pull request merged**